### DISCUSSION TITLE
Boilr keeps crashing on my steam deck when I click images.

### DISCUSSION THREAD

**Discussion by NadaGear** [PhilipK/BoilR#244] [Q&A]
Boilr keeps crashing on my steam deck when I click images. I have uninstalled it from discover to even finding the config files and deleting them and when I reinstall it, it keeps on crashing on me when I click on images.

**Comment by PhilipK** [comment]
This is unfortunately not the first time I have seen this.
It is caused by a bug in the library that reads the images. I keep patching
it, but more bugs show up and the maintained of that library has not merged
my pull request to patch it.
Sorry, rant over.

Here is what you can do.
Find your images grid folder in steam. It is can be found here
".\Steam\userdata\<id3>\config\grid\"

In there, move all the webp images to another folder. Now it should not
crash.

If you want to help fix the problem for good can you zip and upload those
webp images so I can reproduce the bug and debug what is wrong with the
image library?

**Reply by NadaGear** [reply]
where would I put that folder once I find it?

**Reply by NadaGear** [reply]
Thanks, figured it out. Here are the .webp files in question. 

https://mega.nz/file/wdE1yYDb#0JUkQbgLU-Edq4a7rabhbnkg2H2M85I8IhM5wglnqZA
